tests/scanners/test.javascript.js

```diff
@@ -47,6 +67,41 @@ describe('JavaScript Scanner', () => {
     expect(jsScannerWithOptions.options.foo).toEqual('bar');
   });
 
+  it('should be initialised with empty excluded rules object', () => {
+    const jsScanner = new JavaScriptScanner('', 'filename.txt');
+    expect(typeof jsScanner.disabledRules).toEqual('object');
+    // This test assures us the disabledRules can be accessed like an object.
+    expect(typeof jsScanner.disabledRules.someUndefinedProp).toEqual('undefined');
+  });
```

Comment:
I think that test should be more something like

```javascript
it('should have no rules disabled by default', () => {
    const jsScanner = new JavaScriptScanner('', 'filename.txt');
    expect(jsScanner.disabledRules).ToEqual({}); // or [] given my comments above
});
```

That way you can merge those two checks into one.